### ISSUE TITLE
I've ported this to elm 0.19

### ISSUE THREAD

**Issue by bburdette** [periodic/elm-csv#4]
Less of an issue and more of an FYI.  I haven't integrated in the changes from the latest PR, and I haven't gotten elm-test to work on my system because of nixos hijinks.  On the other hand, I ran all the tests by hand and they pass.  

I could make a PR and then maybe someone could get it over the finish line RE the elm-test and the aforementioned merging and testing.  Or I'll do it when I get around to it but not sure when that'll be. 

https://github.com/bburdette/elm-csv/tree/elm-19

**Comment by bburdette** [comment]
I'm stuck on this bit here:  https://github.com/rtfeldman/node-test-runner/issues/226

**Comment by bburdette** [comment]
Yay, tests are passing now, finally got elm-test working.  I got the parseWith thing to work, and the Decoder dealio is next.

**Comment by ericgj** [comment]
Just to let you know, I wrote the Decoder thing, and proposed it as a PR, but didn't hear back for many months. So in the meantime I released it as a [standalone library](https://package.elm-lang.org/packages/ericgj/elm-csv-decode/latest/), and that version has been updated a bit, including getting rid of the infix operators.  So I would propose dropping it from this library when upgrading to 0.19 and leaving it as a separate library, which can work with this or other underlying CSV parsers.
